packages/editor/src/lib/editor/shapes/ShapeUtil.ts

```diff
@@ -604,6 +604,16 @@ export type TLOnBindingChangeHandler<T extends TLShape> = (shape: T) => TLShapeP
 /** @public */
 export type TLOnChildrenChangeHandler<T extends TLShape> = (shape: T) => TLShapePartial[] | void
 
+/** @public */
+export type TLOnHandleCreateHandler<T extends TLShape> = (
+	shape: T,
+	info: {
+		handle: TLHandle
+		isPrecise: boolean
+		initial?: T | undefined
+	}
+) => TLShapePartial<T> | void
+
```

Comment:
I don't see this used anywhere - do we still need it?

Reply:
ah yes, we can lose this